app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,7 @@
+class QuotesController < ApplicationController 
+  def search
```

Comment:
See my comments on your facade. If you implemented something similar to what I showed your controller would then look something like:
```
def search
    @keyword = params[:query]
    @total_count @quotes = QuotableFacade.quote_search(@keyword)
end
```